src/TransCreator/Style.js

```diff
@@ -84,4 +96,23 @@ export default StyleSheet.create({
     fontSize: 20,
     fontWeight: '400',
   },
+  keyboardBtn: {
+    height: DeviceWidth * 0.15,
+    width: DeviceWidth * 0.25,
+    borderRadius: 0,
+    borderColor: 'rgb(240,240,240)',
```

Comment:
Same here, may change it to hex for better maintenance

Reply:
fixed